comps/guardrails/toxicity_detection/toxicity_detection.py

```diff
@@ -0,0 +1,35 @@
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+from langsmith import traceable
```

Comment:
langsmith dependency has been removed from all OPEA comps, pls replace it with latest telemetry interface.